### PR TITLE
Mh tabbed interface

### PR DESCRIPTION
**Pull request by merrillhuang**
Merge tabbed interface branch into main.

### CODE REVIEW

**Review by jessjno — APPROVED** [review]
Looks good to me

**Comment by armstrca** [comment]
Please merge all prior pull requests to main for more efficient code review

**Comment by merrillhuang** [comment]
Will do! Waiting on peer approval for my user profile pull request.

**Comment by merrillhuang** [comment]
All changes from prior branches merged into this branch.

**Review by armstrca — APPROVED** [review]
Slick use of partials!

File: app/controllers/users_controller.rb
Comment (Ritumka) [inline app/controllers/users_controller.rb:2]: I like how you added 'before_action' here!

File: app/views/users/_connections.html.erb
Comment (Ritumka) [inline app/views/users/_connections.html.erb:9]: That's interesting how you make this '_connection.html.erb' partial work both for followers and for following.

File: app/views/users/_tabs.html.erb
Comment (Ritumka) [inline app/views/users/_tabs.html.erb:28]: Nice job with making all this links as partial!

**Review by Ritumka — COMMENTED** [review]
Good job!

**Pull request merged**
